app/views/samples/_info_sample_modal.html.erb

```diff
@@ -0,0 +1,53 @@
+
+<div class="modal fade" id="modal-info-sample" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title"><%= t('samples.modal_info.title', sample: @sample.name) %></h4>
+      </div>
+
+      <div class="modal-body">
+      <% if @sample.my_modules.count > 0 %>
+        <div class="linked-children-datatable">
```

Comment:
Nitpicking, but oh well - it'd make sense to me to also show a text here saying "This sample is assigned to <n> tasks.", similar as how it's shown in protocols modal. Just to be clarify a bit what you're seeing in the table below.